sources/world/Strava-GPS.json

```diff
@@ -0,0 +1,20 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "strava-gps",
+        "name": "Strava GPS traces",
+        "description": "Strava cycling and running heatmap.",
+        "i18n": true,
+        "attribution": {
+            "url": "http://labs.strava.com/heatmap/",
+            "text": "© Strava",
+            "required": true
+        },
+        "overlay": true,
+        "url": "http://heatmap-external-{switch:a,b,c}.strava.com/tiles/all/blue/{zoom}/{x}/{y}.png?px=256",
+        "max_zoom": 16,
```

Comment:
It seems that `max_zoom` is now 12 after https://blog.strava.com/press/heatmap-updates/

Reply:
hi, looks to me like the heatmap now remains visible even when the zoom level is higher that 16, which is a really nice thing, but its quality decreased a lot, it's now pretty much useless to me.
How can we try fix this issue? Strava Slide seems to have the same issue.